### DISCUSSION TITLE
Nextcloud container startup stuck at "System config value files_external_allow_create_new_local set to boolean false"

### DISCUSSION THREAD

**Discussion by fzadikian** [nextcloud/all-in-one#2445] [Questions] [answered]
Hello, I am on a newly deployed EC2 instance and I tried deploying AIO using Docker. Everything went fine until it had to startup the nextcloud-aio-nextcloud container. This is the log:

https://pastebin.com/zZrR1L0t

As you can see, there is no errors. It just got stuck for some reason, and it has been like this for hours. I have tried restarting my containers and even the entire EC2 instance but still, same result. I am quite frustrated because there is not even a single error message to start investigating with. The only clue I have is that the container is unhealthy, as shown here:

`$ sudo docker inspect --format='{{json .State.Health}}' nextcloud-aio-nextcloud
{"Status":"unhealthy","FailingStreak":5,"Log":[{"Start":"2023-05-01T03:50:01.885275737Z","End":"2023-05-01T03:50:01.94399143Z","ExitCode":1,"Output":""},{"Start":"2023-05-01T03:50:31.950599777Z","End":"2023-05-01T03:50:31.999348429Z","ExitCode":1,"Output":""},{"Start":"2023-05-01T03:51:02.008587273Z","End":"2023-05-01T03:51:02.062794697Z","ExitCode":1,"Output":""},{"Start":"2023-05-01T03:51:32.068744773Z","End":"2023-05-01T03:51:32.119271133Z","ExitCode":1,"Output":""},{"Start":"2023-05-01T03:52:02.12382407Z","End":"2023-05-01T03:52:02.183763909Z","ExitCode":1,"Output":""}]}`

Does anybody know what could be happening? Thanks.

**Comment by szaimen** [comment]
Hi, are outgoing connections possible? The container must be able to reach apps.nextcloud.com and github.com. You could try to curl these from inside the container to check if that is possible.

**Reply by fzadikian** [reply]
Everything seems to be fine, curl outputs a whole bunch of stuff for each website (its content) but it never times out or errors out. Ping also works, only thing that does not work is ping6. Although I set up docker for that (step 2 from AIO) I think my VM is not prepared. Could that be an issue?

**Reply by fzadikian** [reply]
It seems that 7hs later, it has advanced a little, but it is still not fully started.
https://pastebin.com/LQhX9f0a

**Reply by szaimen** [reply]
> I think my VM is not prepared. Could that be an issue?

Yes, could be

**Reply by fzadikian** [reply]
> Yes, could be

Should I disable IPv6 on docker or try to get IPv6 working? Would deleting
```
{
  "ipv6": true,
  "fixed-cidr-v6": "2600:1f1e:8c9:4c00::/64",
  "experimental": true,
  "ip6tables": true
}
```
from /etc/docker/daemon.json be enough? Thanks.

**Reply by szaimen** [reply]
> from /etc/docker/daemon.json be enough? Thanks.

yes that should probably work

**Reply by fzadikian** [reply]
> yes that should probably work

Hello, unfortunately that did not work (I restarted the VM make sure changes were applied). The docker container logs show that it is stuck just like in the last log (https://pastebin.com/LQhX9f0a). Could it have to do with my nameserver from Amazon? /etc/resolv.conf shows "172.31.0.2". However I do not think that makes sense since ping and curl actually do work.

**Comment by szaimen** [comment, marked as answer]
I fear I am out of ideas then what could trigger this. Everything works here as expected so I suppose something in your network infrastrucure is not working as expected.

**Reply by fzadikian** [reply]
Thank you, I am trying another method (web installer) to see if I have more luck.
